components/atom/table/src/index.js

```diff
@@ -21,17 +21,24 @@ const AtomTable = ({
   foot,
   fullWidth,
   cellPadding,
-  borderBottom
+  borderBottom,
+  onClick
```

Comment:
If you initialize with a [noop](https://www.damiannicholson.com/posts/2018/03/06/react-noop-anti-pattern/) function it would also always be actionable and you could avoid internal checks.

```suggestion
  onClick = () => {}
```

Reply:
this is not retrocompatible due to styling changes

Reply:
if we set () => {} as a default value for onRowClick prop, the evaluation always will be true and we cannot know when the component is receiving the onRowClick prop or not (for example, on this case when has the callback, the rows have a hover effect)

Reply:
ok, noop then it doesn't work if you are interested in evaluating if you receive the value 👍🏼

Reply:
> this is not retrocompatible due to styling changes

the hover effect is only applied when onRowClick prop is received, and the prop is new too.

Reply:
Of course, i mean. Its not retrocompatible when adding noop as a default value of the mentioned new prop

Reply:
Yes of course, if we use the noop function, the styles will be applied in all cases ;)